graylog2-server/src/main/java/org/graylog2/indexer/indexset/IndexSetConfig.java

```diff
@@ -265,9 +265,11 @@ public static Builder builder() {
 
     @JsonIgnore
     public boolean isRegularIndex() {
+        if (isRegular().isPresent()) {
+            return isRegular().get();
+        }
```

Comment:
I have not idea if it is good that `isWritable()` now can be `false` and `isRegularIndex` can still return `true`...
It seems like the `isRegular()` was introduced just to allow bypass index template checks...